packages/features/ee/users/pages/users-listing-view.tsx

```diff
@@ -2,7 +2,6 @@
 
 import NoSSR from "@calcom/core/components/NoSSR";
```

Comment:
This file is used for `settings/admin/users` page. The layout is handled by app router.